modules/nf-core/angsd/gl/meta.yml

```
@@ -53,11 +59,15 @@ output:
           type: file
           description: File containing genotype likelihoods per sample
           pattern: "*.{glf,beagle}.gz"
+          ontologies:
+            - edam: "http://edamontology.org/format_3750" # Tabular data
```

I think this one is not correct. Do we know if this file is a TSV? If we don't, I would leave it empty
```suggestion
          ontologies: []
```